tools/coqdep_common.ml

```diff
@@ -203,7 +203,7 @@ let error_findlib_name f s =
   Printf.eprintf "in file %s, %s is not a valid plugin name anymore.\n" f s;
   Printf.eprintf "Plugins should be loaded using their public name according to findlib,\n";
   Printf.eprintf "for example package-name.foo and not foo_plugin.\n";
-  Printf.eprintf "If you are building with dune < 2.9.2 you must specify both\n";
+  Printf.eprintf "If you are building with dune < 2.9.4 you must specify both\n";
```

Comment:
Is it confirmed to be fine in 2.9.4 (feature merged upstream etc)?

Reply:
I can take care of that and update the doc in master; open an issue with the right milestone and assign to me if you want. I'm busy until the end of application session.

Reply:
I guess @SkySkimmer wanted a confirmation that .4 was the right one. In any case I hope to purge the compat code before 8.16 is branched (assuming #15560 goes is as well). Then this error message will go.